ifrs17-template/Initialization/InitSystemorphToMemory.ipynb

```diff
@@ -19,9 +19,10 @@
         {
             "cell_type": "markdown",
             "source": [
-                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> In Memory Initialization of unconfigured DataSource </span> </p>",
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> In Memory Initialization of unconfigured DataSource and Transactional Data </span> </p>",
+                "\n<p style=\"font-weight:bold;\"> <span style=\"font-size: 24px\"> DataNode </span> </p>",
                 "\n",
-                "\n<a id='init-to-memory'></a>"
+                "\n<a id='init-base-to-memory'></a>"
```

Comment:
I think this is an anchor. Have you updated the usages? If there aren't any, I suggest to remove it.

Reply:
Indeed. Removed